src/Components/UpcomingTable/upcoming-table.scss

```diff
@@ -1,4 +1,18 @@
 #pagination-options-menu-bottom-bottom-toggle {
     // Not sure why but otherwise the bottom pagination count is displayed in block
     display: flex;
-}
\ No newline at end of file
+}
+
+.drf-upcoming__filter-formgroup {
```

Comment:
The slightly cleaner way to do this would be to have a shared class in a more global stylesheet that both filters could use, given that they're so similar. I think this is fine for now given what we need to deliver before Summit - just want to point this out so you know it can bite you. :)